compiler/p/codegen/PPCSystemLinkage.cpp

```diff
@@ -1299,7 +1299,7 @@ int32_t TR::PPCSystemLinkage::buildArgs(TR::Node *callNode,
                if (!cg()->canClobberNodesRegister(child, 0))
                   {
                   tempRegister = cg()->allocateRegister(TR_VSX_VECTOR);
-                  generateTrg1Src1Instruction(cg(), TR::InstOpCode::xxlor, callNode, tempRegister, argReg);
+                  generateTrg1Src2Instruction(cg(), TR::InstOpCode::xxlor, callNode, tempRegister, tempRegister, argReg);
```

Comment:
the intention looks to me that argRegister is copied into tempRegister. Thus, it should be
            tempRegister = argRegister xxlor argRegister;

Reply:
You're right. Fixed in e255850a3a273d0576ae2c9f3703cab6ca52cd43.